submissions/chabuChabu/css/styles.css

```diff
@@ -0,0 +1,466 @@
+
+}
+
+header ul li a:hover {
+	background-color: #5e578e;
+	transition: 0.5s;
+}
+
+.page-wrapper{
+	min-height: 100%;
+	
+}
+
+/**--Content--**/
+.content{
+	width: 90%;
+	margin: 10px auto 30px;
+}
+
+.content .main-content{
+	width: 58%;
+	float: left;	
+}
+
+.content .main-content .registration-title{
+   
+   font-size: 48px; 
+   color: #69629f;
+   padding-top: 140px;
+   width: 50%;
+   margin: 0 auto;
+  
+}
+
+.content .main-content .section-one{
+	width: 50%;
+	margin: 0 auto;
+	border-radius: 5px;
+	font-size: 25px;
+	color: grey;
+}
+
+.content .main-content .section-one .reg-wording{
+  width: 130%;
+
+}
+
+.content .main-content .section-one .reg-wording a{
+  font-weight: bold;
+
+}
+
+/**Sidebar styles**/
+
+.content .sidebar {
+	width: 30%;
+	float: left;
+	border: 1px solid grey;
+	border-radius: 10px;
+	
+}
+
+.content .sidebar .section {
+	padding: 20px;
+	border-radius: 5px;
+    margin-bottom: 20px;
+}	
+
+.btn{
+	width: 50%;
+	margin: 0 auto;
+	background-color: #69629f;
+	color: #fff;
+	font-weight: bold;
+	padding: .7rem 1rem;
+	border-radius: 5px;
+	display: block;
+	margin: auto;
+	border: none;
+	margin-bottom: 10px;	
+}
+
+#terms{
+	color: #69629f;
+	display: block;
+	padding-top: 15px;
+	text-align: center;
+}
+
+/**Footer**/
+
+.footer{
+	background-color: #69629f;
+	/**padding: 4rem;**/
+	padding: 10px;
+}
+.footer-content{
+	width: 100%;
+	height: 150px;
+	background-color: #69629f;
+	color: white;
+	display: flex;
+	justify-content:space-between;
+	flex-wrap: wrap;
+	margin: 0 auto;
+}
+
+.footer .footer-section a{
+	color: white;
+	text-decoration: none;
+	list-style-type: none;
+
+}
+
+.footer .footer-section ul{
+	list-style-type: none;
+}
+
+.footer .footer-content .footer-section input[type="emailaddress"]{
+    width: 170px;
+	border-radius: 5px;
+	padding: 8px;
+	border: none;
+
+}
+.footer .footer-content .footer-section input[type="button"]{
+    background-color: white;
+	color: #69629f;
+	font-weight: bold;
+    width: 100px;
+	padding: 5px;
+	height: 32px;
+	border: none;
+}
+.footer .footer-content .footer-section input[type="image"]{
+  border-radius: 11px;
+  margin: 8px;
+
+}
+.footer-copyright{
+  background:#69629f;
+  color: white;
+  text-align: center;
+}
+
+
+
+/** end of footer**/
+::placeholder { /* Chrome, Firefox, Opera, Safari 10.1+ */
+  color: #69629f;
+  opacity: 1; /* Firefox */
+}
+
+:-ms-input-placeholder { /* Internet Explorer 10-11 */
+  color: #69629f;
+}
+
+::-ms-input-placeholder { /* Microsoft Edge */
+  color: #69629f;
+}
+
+
+/**Media queries for tablet**/
+
+@media screen and (max-width: 1300px){
+
+	header ul li a{
+	font-size: 16px;
+}
+
+ul .nav{
+	margin-left: 40px;
+	background: yellow;
+	padding:0;
+	list-style: none;
+}
+
+}
+
+@media screen and (max-width: 1300px){
+  .content{
+   width: 100%;
+  }
+  .content .main-content{
+	width: 100%;
+	padding: 10px;
+  }
+
+  .content .main-content .section-one{
+    width: 100%;
+	font-size: 20px;
+	line-height: 0.5cm;
```

Comment:
@ChabuChabu Use px instead of `cm` and `%`  from line 269-272

Reply:
Done